internal/pkg/skuba/addons/metricsserver.go

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright (c) 2020 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"bufio"
+	"encoding/base64"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/kubernetes/cmd/kubeadm/app/images"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"github.com/SUSE/skuba/internal/pkg/skuba/metricsserver"
+	"github.com/SUSE/skuba/internal/pkg/skuba/skuba"
+	skubaconstants "github.com/SUSE/skuba/pkg/skuba"
+)
+
+func init() {
+	registerAddon(kubernetes.MetricsServer, renderMetricsServerTemplate, metricsServerCallbacks{}, normalPriority, []getImageCallback{GetMetricsServerImage})
+}
+
+func GetMetricsServerImage(imageTag string) string {
+	return images.GetGenericImage(skubaconstants.ImageRepository, "metrics-server", imageTag)
+}
+
+func (renderContext renderContext) MetricsServerImage() string {
+	return GetMetricsServerImage(kubernetes.AddonVersionForClusterVersion(kubernetes.MetricsServer, renderContext.config.ClusterVersion).Version)
+}
+
+// CABundle returns base64 encoded Kubernetes CA certificate
+//
```

Comment:
@jenting ,the comment block looks a little detached. Could you remove the blank lines?

Reply:
sure~